src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,36 +2,40 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] numbers = new int[12];
+	public static final int NUMBERS_MAX_SIZE = 12;
+	public static final int IS_EMPTY = -1;
+	public static final int IS_FULL = 11;
+
+	private final int[] numbers = new int[NUMBERS_MAX_SIZE];
 
 	public int getTotal() {
 		return total;
 	}
 
-	private int total = -1;
+	private int total = IS_EMPTY;
 
 	public void countIn(int in) {
 		if (!isFull())
 			numbers[++total] = in;
 	}
 
 	public boolean callCheck() {
-		return total == -1;
+		return total == IS_EMPTY;
 	}
 
 	public boolean isFull() {
-		return total == 11;
+		return total == IS_FULL;
 	}
 
 	protected int peekaboo() {
 		if (callCheck())
-			return -1;
+			return IS_EMPTY;
 		return numbers[total];
 	}
 
 	public int countOut() {
 		if (callCheck())
-			return -1;
+			return IS_EMPTY;
 		return numbers[total--];
```

Comment:
Constant used in callCheck and returned in countOut and peekaboo should have different meaning. That is, you should introduce another constant to e.g. return a default value of rhymer if empty.